test/ParcelRegistry.Tests/AggregateTests/WhenChangingParcelGeometry/GivenParcelExists.cs

```diff
@@ -104,22 +95,14 @@ public void WhenGeometryIsTheSame_ThenNone()
             var caPaKey = Fixture.Create<VbrCaPaKey>();
             var parcelId = ParcelId.CreateFor(caPaKey);
 
-            var extendedWkbGeometry = GeometryHelpers.ValidGmlPolygon.GmlToExtendedWkbGeometry();
```

Comment:
Probably this ValidGmlPolygon is used by default in the builder of both the command and ParcelWasMigrated? This makes it a bit more difficult to understand the test.